src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+import model.Ball;
+
+public class Lottery {
+    public Ball getRandomBall() {

```

Comment:
According to the checklist, avoid creating temporary variables if you can directly return the value. In this case, you can directly return the new Ball instance without storing it in a variable.